### PR TITLE
Update to buster

### PR DESCRIPTION
**Pull request by alexgg**
* balenafin-firmware: Update balenafin overlay
* sd887-mrvl: Update to tag 0.0.5, maintainers and fix for Buster kernels
* Update to buster and add support to build module-only packages

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+ARG DISTRO=buster
+FROM balenalib/rpi-raspbian:${DISTRO}
+
+ARG UID
+ARG GID
 
 RUN apt-get update
 RUN apt-get -y install coreutils quilt parted debootstrap zerofree pxz zip dosfstools bsdtar libcap2-bin grep rsync xz-utils file git curl kmod fakeroot device-tree-compiler dkms
-RUN apt-get -y install build-essential dh-make
+RUN apt-get -y install build-essential dh-make raspberrypi-kernel-headers bc sudo python bison flex libssl-dev
```
Comment (alexgg) [inline Dockerfile]: Review whether all these dependencies are needed now that we are not rebuilding any in-tree module.

File: balenafin-ify.sh [hunk 2]
```diff
@@ -65,13 +67,19 @@ log "Importing bintray gpg key..."
 curl -fsSL 'https://bintray.com/user/downloadSubjectPublicKey?username=bintray' | apt-key add -
 
 log "Adding balenaFin Raspbian repository..."
-echo "deb https://dl.bintray.com/balenaos/raspbian stretch main" > /etc/apt/sources.list.d/balenafin.list
+echo "deb https://dl.bintray.com/balenaos/raspbian ${DISTRO} main" > /etc/apt/sources.list.d/balenafin.list
 
 log "Installing required packages..."
 apt-get -y install apt-transport-https
 apt-get -y update
```
Comment (chriswoj) [inline balenafin-ify.sh:73]: On a vanilla raspbian OS the apt-get update should be done before the first install. 

```
pi@raspberrypi:~/balenafin-raspbian $ sudo ./balenafin-ify.sh
[LOG][balenafin-ify.sh] Importing bintray gpg key...
OK
[LOG][balenafin-ify.sh] Adding balenaFin Raspbian repository...
[LOG][balenafin-ify.sh] Installing required packages...
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following NEW packages will be installed:
  apt-transport-https
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 149 kB of archives.
After this operation, 156 kB of additional disk space will be used.
Err:1 http://raspbian.raspberrypi.org/raspbian buster/main armhf apt-transport-https all 1.8.2.1
  404  Not Found [IP: 93.93.128.193 80]
E: Failed to fetch http://raspbian.raspberrypi.org/raspbian/pool/main/a/apt/apt-transport-https_1.8.2.1_all.deb  404  Not Found [IP: 93                                                                                                                                                                                      .93.128.193 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

Comment (alexgg) [reply balenafin-ify.sh:73]: Thanks for the feedback, I have now updated the PR.

**Comment by alexgg** [comment]
@balena-ci retest

**Comment by alexgg** [comment]
@balena-ci retest

**Review by majorz — APPROVED** [review]

**Comment by alexgg** [comment]
@balena-ci retest

**Comment by nazrhom** [comment]
@balena-ci retest

**Pull request merged**